src/main/resources/jupyter/init-actions.sh

```diff
@@ -171,13 +178,17 @@ if [[ "${ROLE}" == 'Master' ]]; then
         log 'Copying SA into RStudio Docker...'
         docker cp /etc/${SERVICE_ACCOUNT_CREDENTIALS} ${RSTUDIO_SERVER_NAME}:/etc/${SERVICE_ACCOUNT_CREDENTIALS}
       fi
+      if [ ! -z ${WELDER_DOCKER_IMAGE} ] && [ "${WELDER_ENABLED}" == "true" ] ; then
```

Comment:
does this mean we're copying service account file to welder container?

Reply:
Only if when we populate `$SERVICE_ACCOUNT_CREDENTIALS` - which is not something we normally do.  Maybe some comments would help clarify here.

Reply:
hey @rtitle , can you comment on whether we need this code if welder is just using application default credential?

Reply:
I think yes.

In the normal case (we do not set SERVICE_ACCOUNT_CREDENTIALS) then this does nothing.

In the weird case (some kind of pluggable auth where we do in fact set SERVICE_ACCOUNT_CREDENTIALS) then this is the code which enables Google Application Default Credentials to be equal to SERVICE_ACCOUNT_CREDENTIALS, which is necessary for anything using the defaults.

Since Welder will be using ADC (like Jupyter does) then it will have the correct credentials in both cases